techniques/css/C29.html

```diff
@@ -396,28 +392,28 @@ <h5>In &lt;body&gt; section</h5>
 
 
 			<section id="the-css-components">
-				<h4>The CSS components</h4>
+				<h4>CSS コンポーネント</h4>
 				<section id="style1-css">
-					<h5>In style1.css</h5>
+					<h5>style1.css</h5>
 			
 <pre xml:space="preserve"><code class="language-css">body, p { color:#000000; background-color:#FFFFFF; }
 h1, h2, h3 {color:#990000; background-color:#FFFFFF; }</code></pre>
 				</section>
 				<section id="style2-css">
 
-					<h5>In style2.css</h5>
+					<h5>style2.css</h5>
 <pre xml:space="preserve"><code class="language-css">body, h1, h2, h3, p, a { color:#FFFF00; background-color:#000000; }</code></pre>
 				</section>
 			</section>
 		</section>
-			<p class="working-example">A working example of this code, <a href="../../working-examples/uri-parameter-choose-stylesheet/">Using PHP $_GET to apply a different external CSS file</a>, is available.</p>
+			<p class="working-example">このコードの実装例 <a href="../../working-examples/uri-parameter-choose-stylesheet/">Using PHP $_GET to apply a different external CSS file</a> が利用可能である。</p>
```

Comment:
working example の訳を、他のテクニックと合わせて「動作例」としました。
```suggestion
			<p class="working-example">このコードの動作例 <a href="../../working-examples/uri-parameter-choose-stylesheet/">Using PHP $_GET to apply a different external CSS file</a> が利用可能である。</p>
```